packages/react-dialog/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@fluentui/react-dialog",
+  "version": "9.0.0-beta.5",
```

Comment:
This should be `alpha`, right?

Reply:
Unfortunately I think we have to use the same prerelease tag on everything right now. @ling1726 might remember more of the details.

Reply:
Confirmed with @ling1726 that this version is correct one as we can only support a single pre-release tag.